chromatic_fitting.py

```diff
@@ -78,127 +86,277 @@ def summarize_parameters(self):
         for k, v in self.parameters.items():
             print(f"{k} =\n  {v}\n")
 
+    def reinitialize_parameters(self, exclude=[]):
```

Comment:
Just to be clear, this is keeping the original parameter objects that got created before, but just deleting the pymc3_model associated with them, right?

Reply:
Yes exactly